standards/variables.rst

```diff
@@ -41,6 +43,6 @@ A variable name should adhere to the following conventions:
 Units
 -----
 
-The **unit** attribute is **required** and its value should be compatible with the
-`iam-units <https://github.com/iamconsortium/units>`_ package. Alternatively,
-a variable can be *dimensionless*, i.e., not have a unit.
+The **unit** associated with a variable should be compatible with the
+`iam-units <https://github.com/iamconsortium/units>`_ package.
+Alternatively, a variable can be *dimensionless*, i.e., not have a unit.
```

Comment:
Maybe an example of a dimensionless unit would be nice.

Reply:
Added the Human Development Index as an example.